### ISSUE TITLE
Deleted `yarn.lock` in stake/packages/frontend

### ISSUE THREAD

**Issue by 0xjjpa** [hoprnet/hopr-devrel#155]
_tl;dr in #150 we deleted the FE `yarn.lock` of the staking UI and added a blank (?) one instead. Restore it in `packages/frontend` and remove it from the root directory._

Seems like because `hopr-devrel` is a monorepo of other monorepo's (e.g. `stake` has two packages, `hopr-stake` and `frontend`), `yarn.lock`s can be often misplaced. This is the case for `stake/packages/frontend`, where the `yarn.lock` file was deleted in #150.

![image](https://user-images.githubusercontent.com/1128312/149530637-3a72b8e8-7c79-4bf2-b6f2-12ccaa04a486.png)

Here's how it should look, as seen in [main](https://github.com/hoprnet/hopr-devrel/blob/main/stake/packages/frontend/yarn.lock).

![image](https://user-images.githubusercontent.com/1128312/149531030-def33960-3df6-43e0-a229-1835bd9bae43.png)

Here's it currently missing in `release/stake-v1`

![image](https://user-images.githubusercontent.com/1128312/149531126-9300c6b6-6b42-4cdb-8016-87644d4f086b.png)

and here's the empty one in the root directory (i.e it shouldn't be there).

![image](https://user-images.githubusercontent.com/1128312/149531262-44525aa4-cc05-4a62-a5b6-486f160f69b3.png)
